sdk/appcenter-distribute/src/test/java/com/microsoft/appcenter/distribute/DistributeWarnUnknownSourcesTest.java

```diff
@@ -379,14 +379,14 @@ public void disableThenClickSettingsThenFailsToNavigate() throws Exception {
         /* Click settings. */
         Intent intent = mock(Intent.class);
         whenNew(Intent.class).withArguments(Settings.ACTION_SECURITY_SETTINGS).thenReturn(intent);
-        doThrow(new ActivityNotFoundException()).when(mFirstActivity).startActivity(intent);
+        doThrow(new ActivityNotFoundException()).when(mContext).startActivity(intent);
         ArgumentCaptor<DialogInterface.OnClickListener> clickListener = ArgumentCaptor.forClass(DialogInterface.OnClickListener.class);
         verify(mDialogBuilder).setPositiveButton(eq(R.string.appcenter_distribute_unknown_sources_dialog_settings), clickListener.capture());
         clickListener.getValue().onClick(mUnknownSourcesDialog, DialogInterface.BUTTON_POSITIVE);
         when(mUnknownSourcesDialog.isShowing()).thenReturn(false);
 
         /* Verify navigation attempted. */
-        verify(mFirstActivity).startActivity(intent);
+        verify(mContext).startActivity(intent);
```

Comment:
Do we need to mention about this fix to the changelog?

Reply:
Add update to changelog